src/commands/node.mjs

```diff
@@ -896,6 +924,203 @@ export class NodeCommand extends BaseCommand {
     return true
   }
 
+  async refresh (argv) {
+    const self = this
+
+    const tasks = new Listr([
+      {
+        title: 'Initialize',
+        task: async (ctx, task) => {
+          self.configManager.update(argv)
+          await prompts.execute(task, self.configManager, [
+            flags.namespace,
+            flags.nodeIDs,
+            flags.releaseTag,
+            flags.cacheDir,
+            flags.keyFormat
+          ])
+
+          const config = {
+            namespace: self.configManager.getFlag(flags.namespace),
+            nodeIds: helpers.parseNodeIds(self.configManager.getFlag(flags.nodeIDs)),
+            releaseTag: self.configManager.getFlag(flags.releaseTag),
+            cacheDir: self.configManager.getFlag(flags.cacheDir),
+            force: self.configManager.getFlag(flags.force),
+            applicationEnv: self.configManager.getFlag(flags.applicationEnv),
+            keyFormat: self.configManager.getFlag(flags.keyFormat),
+            de3vMode: self.configManager.getFlag(flags.devMode),
```

Comment:
Is there any reason to call this variable `de3vMode`?

Reply:
that was an accidental keystroke.  Since you are out, I'll merge this and create a small PR to fix it.